Fix/existing secret key (#202)
treeverse/charts: charts/lakefs/README.md

| `secrets.databaseConnectionString`    | PostgreSQL connection string to be used by lakeFS. (Ignored when existingSecret is set)                                                                                                                                                                                              |                                     |
| `secrets.authEncryptSecretKey`        | A random (cryptographically safe) generated string that is used for encryption and HMAC signing. (Ignored when existingSecret is set)                                                                                                                                                |                                     |
| `existingSecret`                      | Name of existing secret to use for the chart's secrets (by default the charts create a secret to hold the authEncryptSecretKey and databaseConnectionString                                                                                                                          |                                     |

> | `secretKeys.databaseConnectionString` | Name of key in existing secret to use for databaseConnectionString (no default). Only used when existingSecret is set and PostgreSQL is used.                                                                                                                                        |                                     |

Reviewer: ```suggestion
| `secretKeys.databaseConnectionString` | Name of key in existing secret to use for a PostgreSQL databaseConnectionString (no default). Only used when sed when `existingSecret is set`                                                                                                                                        |                                     |
```
